src/log_impl.rs

```diff
@@ -571,6 +681,7 @@ enum LogError {
     Send(mpsc::SendError<String>),
     #[cfg(feature = "syslog-4")]
     Syslog4(syslog_4::Error),
+    CustomLogFile(String)
```

Comment:
If we keep this error:

Since this is error is only ever used once, and always with the same string, what would you think of making this a `CannotOpenFile` variant? We could hardcode the error message in the `Display` implementation.

The way I'm seeing it this `String` is currently entirely redundant - it's always the same static string we know, so we could eliminate the variable entirely.